tests_e2e/pipeline/pipeline.yml

```diff
@@ -9,7 +9,7 @@ parameters:
   - name: test_suites
     displayName: Test Suites
     type: string
-    default: agent_bvt
+    default: agent_bvt,agent_update
```

Comment:
Added for daily monitoring agent update. After this merge, I'll schedule daily run for agent_update